### PR TITLE
Website implementation

### PR DESCRIPTION
**Pull request by Neb19**
Set the base of website. The files Html, Css and Javascript were modified.

### CODE REVIEW

**Comment by pimolo** [comment]
286 files changed :octocat: 
Most of them are assets files that you didn't change by yourself.
Actually, in most projects, we don't care about assets — developers don't modify them.
That's why this kind of files should not be committed (or at least it should be avoided each time you can).
Here, you use Bower to manage your assets. It means that any contributor can fetch these files, simply executing `bower install` (if `bower.json` is well committed). So versionning them is useless and it can even create problems later.

So, what you just have to do is execute `git rm www/assets`, and then add `www/assets/` to the `.gitignore` file. It will tell Git to stop watching assets.

TL;DR:
Generally, DO NOT commit files you don't modified yourself, and find a solution to fetch them externally to the repository.

**Comment by pimolo** [comment]
Additionnaly you committed some conflicts :D
